src/telemetry/telemetry.ts

```diff
@@ -0,0 +1,35 @@
+export type ErrorEvent = Readonly<{
+	kind: 'failedToExecuteCommand';
+	commandName: string;
+	errorMessage: string;
```

Comment:
can error message contain file paths or other sensitive information?

Reply:
we do not include stack so its more or less safe. but i think we cannot guarantee that some thirdparty lib does not include file name in the message. Do you propose to exclude errorMessage from tracking? Or add some kind of message validation?